### ISSUE TITLE
agetty fails to load with --noclear option

### ISSUE THREAD

**Issue by eNTi** [OpenRC/openrc#217]
Since the update to 0.35.2 (on Artix) agetty fails to load if it's given the --noclear option. i've not tried any other options so far.

/etc/conf.d/agetty.tty1:
```
# extra options to pass to agetty for this port
agetty_options="--noclear"
```

**Comment by williamh** [comment]
Can you try with 0.35.3 or even better 0.35.4?

**Comment by williamh** [comment]
There was an off-by-one error that was fixed in 0.35.5 that should resolve this.
Please let me know if it doesn't and I'll re-open this issue.

Thanks,

William
